game/sprite-backend.rpy

```diff
@@ -1082,9 +1086,57 @@ image natsuki idle = ConditionSwitch(
     predict_all = True
 )
 
+image natsuki test = ConditionSwitch(
+    "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsAbove()", "natsuki 5utlsml",
+    "Natsuki.getMouseIsRight() and Natsuki.getMouseIsAbove()", "natsuki 5utrsml",
+    "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsBelow()", "natsuki 5udlsml",
+    "Natsuki.getMouseIsRight() and Natsuki.getMouseIsBelow()", "natsuki 5udrsml",
+    "Natsuki.getMouseIsLeft()", "natsuki 5ullsml",
+    "Natsuki.getMouseIsRight()", "natsuki 5ulrsml",
+    "Natsuki.getMouseIsAbove()", "natsuki 5uupsml",
+    "Natsuki.getMouseIsBelow()", "natsuki 5udwsml",
+    "True", "natsuki 5unmsml",
+    predict_all = True
+)
+
+label dynatest:
+    show natsuki test
+    n ""
+    return
+
 # Idle images for ENAMORED+
 image natsuki idle enamored:
     block:
+        choice:
+            ConditionSwitch(
+                "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsAbove()", "natsuki 5utlsml",
+                "Natsuki.getMouseIsRight() and Natsuki.getMouseIsAbove()", "natsuki 5utrsml",
+                "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsBelow()", "natsuki 5udlsml",
+                "Natsuki.getMouseIsRight() and Natsuki.getMouseIsBelow()", "natsuki 5udrsml",
+                "Natsuki.getMouseIsLeft()", "natsuki 5ullsml",
+                "Natsuki.getMouseIsRight()", "natsuki 5ulrsml",
+                "Natsuki.getMouseIsAbove()", "natsuki 5uupsml",
+                "Natsuki.getMouseIsBelow()", "natsuki 5udwsml",
+                "True", "natsuki 5unmsml",
+                predict_all = True
+            )
+            pause 4
+            "natsuki 3fchbll"
+            pause 1
+            "natsuki 3fchsml"
+            pause 3
+
+        choice:
+            ConditionSwitch(
+                "Natsuki.getMouseIsLeft()", "natsuki 7tslsml",
+                "Natsuki.getMouseIsRight()", "natsuki 7tsrsml",
+                "True", "natsuki 7tsqsml",
+                predict_all = True
+            )
+            pause 4
+            "natsuki 7fchsml"
+            pause 3
```

Comment:
Also wanted to add it is very clever to have different variants to keep it interesting!